src/librustdoc/html/static/js/search.js

```diff
@@ -4408,17 +4408,18 @@ ${item.displayPath}<span class="${type}">${name}</span>\
             }
         });
     } else if (query.error === null) {
+        const dlroChannel = `https://doc.rust-lang.org/${getVar("channel")}`;
```

Comment:
To fix the CI error here, you need to add getVar to the globals list at the top of the file. It should be available to this file. cc @GuillaumeGomez  -- correct me if I'm wrong